test/integration.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# Copyright 2017 The Kubernetes Authors.
```

Comment:
I would prefer the way @MHBauer has this now over the script @duglin has suggested.

While I'm not really a huge fan of promoting different permutations of the build process and agree that out-of-container build / test should be done at one's own risk, it's pretty obvious that this script would break things pretty badly for anyone who elects to do out-of-container build / test...

Let's assume you don't have docker installed or simply don't want to use it-- or _can't_ (this would apply to Google's Jenkins server that uses Dockerized slaves and can't do D-in-D). Relying on this script (which in-turn relies on Docker) to facilitate integration testing would mean that anyone in the circumstances I just noted simply cannot run those tests. Jenkins unable to run integration tests? Might be a problem.

The beauty of how @MHBauer has things now (etcd bundled into our development image) is that testing in-container or out makes the same core assumption-- that you can talk to an etcd at localhost:2379. If you've elected to run out-of-container, you siply bring your own etcd to the party, which is more tenable than requiring that you use Docker-- the very technology you were trying to avoid by running out-of-container in the first place.